src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -83,6 +84,9 @@
     private boolean hasUpgrade;
     private Correlation<ServerAcceptState> correlation;
     private boolean targetBeginIssued;
+    private Runnable cleanupConnectReply = () -> {};
```

Comment:
No need to default, if logic updated below for reaction to accept request `ABORT`.